sig-network/README.md

```diff
@@ -31,7 +31,7 @@ The [charter](charter.md) defines the scope and governance of the Network Specia
 * SIG Network Multi-Network Meeting: [Wednesdays at 08:00 PT (Pacific Time)](https://zoom.us/j/95680858961?pwd=M1c2TTdMZHpMUUtIYXRpbjRobkNJZz09) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=08:00&tz=PT%20%28Pacific%20Time%29).
   * [Meeting notes and Agenda](https://docs.google.com/document/d/1pe_0aOsI35BEsQJ-FhFH9Z_pWQcU2uqwAnOx2NIx6OY/edit).
   * [Meeting recordings](https://www.youtube.com/watch?v=phCA5-vWkVM&list=PL69nYSiGNLP2E8vmnqo5MwPOY25sDWIxb).
-* Sig-Network and Gateway API Code Jam: [Fridays at 08:30 PT (Pacific Time)](https://zoom.us/j/96900767253) (Last Friday of the Month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=08:30&tz=PT%20%28Pacific%20Time%29).
+* Sig-Network Jam: [Fridays at 08:30 PT (Pacific Time)](https://zoom.us/j/96900767253) (Last Friday of the Month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=08:30&tz=PT%20%28Pacific%20Time%29).
```

Comment:
```suggestion
* Sig-Network Code Jam: [Fridays at 08:30 PT (Pacific Time)](https://zoom.us/j/96900767253) (Last Friday of the Month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=08:30&tz=PT%20%28Pacific%20Time%29).
```

Reply:
sorry, meant to keep the "Code Jam" part I think that's an important part of its current identity

Reply:
ya definetly